### DISCUSSION TITLE
Add a twitter bot to announce new releases

### DISCUSSION THREAD

**Discussion by kordano** [replikativ/datahike#485] [Ideas]
Let's have a twitter bot to announce new releases and PRs.

**Comment by kordano** [comment]
Thanks @elMor3no for the idea.

**Comment by TimoKramer** [comment]
well and slack obviously :)

**Comment by kordano** [comment]
Yes of course. And we have to discuss if we would still support our discord channel. Too many platforms for discussions and feedback are somewhat confusing.

**Comment by TimoKramer** [comment]
Did it but still tweeting on my personal account. Should be lambdaforge in the future.

**Comment by Ramblurr** [comment]
On the topic of release announcements, I make regular use of the `Watch -> Custom Events -> Releases` feature of GitHub. It's quite handy.

I would like to see the [CHANGELOG](https://github.com/replikativ/datahike/blob/main/CHANGELOG.md) updated regularly.. there's a big discrepancy between it and the current GitHub Releases (0.4 vs 0.6.xx)

**Reply by TimoKramer** [reply]
Hi @Ramblurr, where would you like to see announcements? Twitter still important for you?

**Reply by Ramblurr** [reply]
Hi @TimoKramer ! For me the Github Release is sufficient..it shows up in my Github notifications dashboard. The pain point I have with releases at the moment is that the Changelogs are not grokkable. As a user of datahike who is not too familiar with the internals, it would be very useful to have a user-facing description of what changed in the releases. 

In the case that the changes don't directly affect me as a user (in some way I should care about with respect to features, upgrading,  or performance), then a vague description is sufficient, for example: "Various internal changes".


![image](https://user-images.githubusercontent.com/14830/203523342-9134e15c-fdb6-482a-b299-939049180a75.png)

![image](https://user-images.githubusercontent.com/14830/203524257-206a5d08-67d2-4c72-a96e-41167e5f90ac.png)

**Reply by TimoKramer** [reply]
you are absolutely right. since the new release model we let things slide in terms of changelog. We will make a proper release soon with a proper changelog and think about how to improve keeping up with it. I really want to see ADRs as well but that is another topic. But documentation is key.

**Reply by TimoKramer** [reply]
We've discussed changelogs and we decided that we won't write changelogs anymore since we are keeping an extra effort to write proper commits and the Github-releases are showing what's changed anyway. I will switch the release pipeline to always create a release without any extra click.

**Reply by Ramblurr** [reply]
That sounds fine to me.

However if there are breaking changes, or important upgrade information, will that be communicated someway outside of the git commits?

**Reply by TimoKramer** [reply]
We will announce these things in our release-notes on github for the time being. It's just a matter of discipline.

**Comment by TimoKramer** [comment]
We have a bot now but Twitter is not relevant anymore
